src/pages/accessControl.ts

```diff
@@ -26,8 +26,10 @@ export function canAccessPage(pageName: PageName): boolean {
 			return !!asc.state.login.permissions[PERMISSIONS.PERM_UPDATE_DONATION_FUNDS]
 		case PageName.MANAGE_ACCESS:
 			return !!asc.state.login.permissions[PERMISSIONS.PERM_MANAGE_ACCESS]
+		case PageName.SIGNOUTS_TABLES:
+			return true; 
 		default:
-			assertNever(pageName);
+			//assertNever(pageName);
```

Comment:
The purpose of the assertNever is to throw a compile time error if you are missing a page in this switch.  As long as you've got all the pages in there, this should be fine to leave in